.eslintrc.json

```diff
@@ -40,6 +42,9 @@
         "beforeEach": "readonly"
     },
     "rules": {
+        "formatjs/enforce-default-message": "error",
+        "formatjs/enforce-description": "error",
+        "formatjs/enforce-id": "error",
```

Comment:
Can we switch the sorting order on this? I under stand that this order is laying out the messages attributes in alphabetical order but it makes the file harder to read through. I think having the messages themselves in alphabetical order is a good thing but the attributes should be in the following order for easier readability.

 id: '',
description: '',
defaultMessage: '',